### ISSUE TITLE
Being an operative with a family heirloom leaves the heirloom on the title screen zone

### ISSUE THREAD

**Issue by coiax** [tgstation/tgstation#42189]
I was admining on Round 99651, and made the title screen invisible briefly to demonstrate what it looked like without it, and found a lighter just hanging out in the single free turf.

Turns out it was one of the operative's heirloom lighters.

This is pretty silly, I will look into seeing how this could occur.

**Comment by SpaceManiac** [comment]
I ran into this before. It happens to not be too big an issue because becoming an operative (or Wizard) clears your quirks.

1. Minds are initialized.
2. The game mode selects antags. Those who will spawn off-station get their `assigned_role` and `special_role` set ("Nuclear Operative", "Wizard").
3. Everyone without an `assigned_role` is assigned jobs.
4. Everyone's bodies are created in the start area.
5. Everyone whose `assigned_role` is not their `special_role` is given their job equipment, which includes moving them to their spawnpoint.
6. Everyone's quirks are assigned.
7. Everyone's clients are assigned to their characters.
8. The round is considered started.
9. The game mode assigns antag datums to the previously selected minds. For nukeops and wizard, this is the step where they are moved to their spawnpoint.

As can be seen, special roles have their quirks assigned before they are moved to their spawnpoints or given their equipment, and without pockets or a backpack the heirloom is placed on the ground in the start area. Operative proc is `/datum/controller/subsystem/ticker/proc/equip_characters()`.

**Comment by kriskog** [comment]
Wizards start with the heirloom beneath them in the den

**Comment by SpaceManiac** [comment]
Looks like wizard `new_player` mobs are moved to the den before body creation. The same is not true of nuclear operatives.
```dm
/datum/game_mode/wizard/pre_setup()
	/*...*/
	for(var/datum/mind/wiz in wizards)
		wiz.current.forceMove(pick(GLOB.wizardstart))
```

**Comment by kriskog** [comment]
Should be fixed by #44781
